features/storage/csi_clone.feature

```diff
@@ -0,0 +1,67 @@
+Feature: CSI clone testing related feature
+  """
+  This test can only work on a Cluster which has CSI driver deployed, OSP has csi deploy by default from 4.7. 
+  When deploy a cluster which has no csi deployed, please use enable_tpnu_featureset: "yes" in LAUNCHER_VARS.
+  From 4.7, OSP CSI driver was installed by default. Its storage class name is standard-csi.
+  Use below command can temporarily add cs class(oc get sc, can find all deployed stroage class):
+  oc patch featuregate cluster -p '{"spec": {"featureSet": "TechPreviewNoUpgrade"}}' --type merge
+  """
+  
+  # @author jianl@redhat.com
+  # @case_id OCP-27615
+  @admin
+  Scenario: Clone a pvc on OSP in the same namespace is successfully
+    Given the master version >= "4.8"
+    Given I switch to cluster admin pseudo user
+    # Step 1
+    # From 4.7, OSP CSI driver was installed by default. The storage class name is standard-csi
+    # Step 2
+    When admin creates a project with:
+      | project_name | clone |
+    Then the step should succeed
+    Given I use the "clone" project
```

Comment:
Usually we use the <random> project name instead of hard coding it, it will avoid the conflict with other cases which happen to use the same project, or we can use "clone-<random>" if you want to make it meaningful.

Reply:
Good suggestion. I did not know that step before...

Reply:
Sorry I notice that github remove some word in my comment, I mean to use the project with a random name, or clone+random name.
The step will give you a project with a random name
> Given I have a project

Reply:
The step `Given I have a project` will do, in most cases.

Reply:
@jhou1 Yes, i am trying it now.